test/mapmatch_config.cc

```diff
@@ -0,0 +1,146 @@
+#include "baldr/rapidjson_utils.h"
+#include <boost/property_tree/ptree.hpp>
+
+#include "meili/config.h"
+
+#include "test.h"
+
+namespace {
+
+boost::property_tree::ptree json_to_pt(const std::string& json) {
```

Comment:
ACK! We have [12 places](https://github.com/valhalla/valhalla/search?q=%22ptree+json_to_pt%22&unscoped_q=%22ptree+json_to_pt%22) where this function is copied. We should add it to a header somewhere and not have 13 places. 

See also: https://github.com/valhalla/valhalla/pull/2481#discussion_r460284761

/cc @PureW

Reply:
good catch! will fix